hypothesis-python/tests/numpy/test_argument_validation.py

```diff
@@ -267,6 +267,7 @@ def e(a, **kwargs):
         e(nps.basic_indices, shape=(0, 0), max_dims=-1),
         e(nps.basic_indices, shape=(0, 0), max_dims=1.0),
         e(nps.basic_indices, shape=(0, 0), min_dims=2, max_dims=1),
+        e(nps.basic_indices, shape=(0, 0), min_dims=50),
```

Comment:
It's a fine test case, it just already exists on the next line!

Reply:
Ahh so I figured out that I explicitly added this because of coverage, as I use the `@check_function`-wrapped `check_valid_dims` for `min_dims`. I believe it makes sense to specifically tell the user their `min_dims` is too large, and so I think this test case should stay.

Incase folk missed it, this test case is for *min* dims, not max like the case below.

Reply:
> Incase folk missed it, this test case is for _min_ dims, not max like the case below.

Ahem.  Yes, that's useful 🤦‍♂️

Reply:
Oh wow. Yeah I misread that like three times. Sorry!